ui/receiver/receiver.go

```diff
@@ -174,12 +176,17 @@ func (m model) Update(msg tea.Msg) (tea.Model, tea.Cmd) {
 			time.Since(m.transferProgress.TransferStartTime).Round(time.Millisecond).String(),
 			ui.ByteCountSI(m.transferProgress.TransferSpeedEstimateBps),
 		)
+
+		filetable.WithMaxHeight(math.MaxInt)(&m.fileTable)
+		m.fileTable = m.fileTable.Finalize().(filetable.Model)
 		return m, ui.TaskCmd(message, tea.Batch(m.spinner.Tick, decompressCmd(msg.temp)))
 
 	case decompressionDoneMsg:
 		m.state = showFinished
 		m.receivedFiles = msg.filenames
 		m.decompressedPayloadSize = msg.decompressedPayloadSize
+
+		filetable.WithFiles(m.receivedFiles)(&m.fileTable)
```

Comment:
Would be nicer to expose a `SetFiles` method on the table, which then can be called in the option. Instead of calling the option after creating the table.